packages/traceability/issuer-api/src/pods/options/types.ts

```diff
@@ -0,0 +1,21 @@
+import { Signer } from 'ethers';
+
+export type SignerAdapter = (blockchainNetId: number) => Promise<Signer>;
+
+export interface IssuerModuleOptions {
+    enableTransactionLogging: boolean;
+    enableCertificationRequest: boolean;
+    getOperatorEncryptionKey?: () => string;
```

Comment:
Why not:
1. Create a `EncryptedAdapter`
2. Require `EncryptedAdapter` to have encryption key set
2. Rename `customSignerAdapter` to `signerAdapter`
3. Set `EncryptedAdapter` as the default `signerAdapter`?

Having these two issuer module properties is redundant, since they relate to the same functionality.

Reply:
Well, yeah, more or less that would be a good idea.
I had some problem with this approach initially, but I now just figured out how to resolve this.

Reply:
IssuerModule shouldn't really care about the encryption key, only the EncryptedPrivateKeyAdapter  adapter.